pom.xml

```diff
@@ -126,5 +119,79 @@
         <hadoop.version>${hadoop3.version}</hadoop.version>
       </properties>
     </profile>
+    <profile>
+      <!-- default HBase profile -->
+      <id>hbase2</id>
+      <activation>
+        <property>
+          <name>!hbase.profile</name>
```

Comment:
It would be nice if -Dhbase.profile=2 worked and was the default. Similarly, -Dhadoop.profile=3 is the default but specifying it works.